src/main/java/hu/wurfel/new_school_reference/subject/SubjectDto.java

```diff
@@ -0,0 +1,23 @@
+package hu.wurfel.new_school_reference.subject;
+
+import lombok.Data;
+
+@Data
+public class SubjectDto {
+
+	private Long id;
+	private String name;
+
+	public boolean hasId() {
+		return id != null;
+	}
+
+	public boolean hasName() {
+		return name != null;
+	}
+
+	public boolean isEmpty() {
+		return id == null
+				&& name == null;
```

Comment:
is

Ha csak 2 érték van, akkor talán felesleges a sortörés, de alapvetően nem gond. Ha átírod methodokra, akkor viszont egészen nagyon felesleges.

Reply:
megint
átírtam és egy sorba kerültek